rotkehlchen/accounting/accountant.py

```diff
@@ -320,6 +323,7 @@ def process_history(
         self.eth_transactions_gas_costs = FVal(0)
         self.asset_movement_fees = FVal(0)
         self.csvexporter.reset()
+        self.csvexporter.add_report(start_ts, end_ts)
```

Comment:
here shouldn't it be `self.csvexporter.report_id = self.csvexporter.add_report(start_ts, end_ts)` ?

Reply:
I have the CSV exporter managing its report id. Reset initializes a new report, then as process history runs, recent events use that report_id as a foreign key.

Here, CSV exporter sets one report id per instance and has no return in add_report

```
def add_report(self, start_ts: Timestamp, end_ts: Timestamp) -> None:
        self.report_id = self.cache.add_report(start_ts, end_ts)
```

however, the accountant class can still access the report id as you mentioned by reading self.csvexporter.report_id